### PR TITLE
feat(login): Added login view, butterknife, viewModel and splash Screen

### PR DESCRIPTION
**Pull request by durini309**

### CODE REVIEW

File: features/login/build.gradle [hunk 1]
```diff
@@ -1,6 +1,5 @@
 apply plugin: 'com.android.library'
-apply plugin: 'kotlin-android-extensions'
-apply plugin: 'kotlin-android'
+apply plugin: 'com.jakewharton.butterknife'
```
Comment (cancinos) [inline features/login/build.gradle]: Is it necessary to call `` apply plugin `` everytime we want to inject butterknife dependency on a feature module?

**Comment by cancinos** [comment]
@durini309 Could you please provide a little description about this merge request, please?

File: features/login/build.gradle (hunk 1 above)
Comment (durini309) [reply features/login/build.gradle]: Yes it is necessary. You could find more info here: https://github.com/JakeWharton/butterknife#library-projects

**Comment by durini309** [comment]
So, this merge contains this changes:
* Login UI. The only thing that remains is the UI for the landscape view
* Added Butterknife: Butterknife is a dependency injection library help us writing less boilerplate when we reference an element for the view. [Read more here](https://github.com/JakeWharton/butterknife)
* ViewModel: this is used for the architecture MVVM. You arleady know what it is, so I won't explain anymore.
* Splash Screen: This screen is the first thing shown from the app when the app was not in RAM. So, before this change, it showed a purple toolbar with a blank activity. Now, it shows a completely white screen. In the future we should add a view with our logo or something like that.

**Pull request merged**